### PR TITLE
DOCS-7457 Enhancing Developer Workflows with Datadog Update

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Adds a section for creating a Jira issue through the Case Management integration.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/tests/developer_workflows.md [hunk 1]
```diff
@@ -122,6 +126,38 @@ You can enable test summaries in pull requests with the following steps:
 
 Comments only appear on pull requests that were opened before the test run and that have run at least one test for an enabled test service.
 
+## Create Jira issues
+
+With [Case Management][7], you can create and open pre-filled Jira issues with relevant context into your tests as well as deep links back to Datadog for more streamlined debugging workflows. Creating issues directly from Test Visibility can help you track and maintain accountability for test failures and flaky tests.
+
+### In-app entry points
+
+You can create Jira issues through Case Management from three areas within Test Visibility:
```
Comment (alai97) [inline content/en/tests/developer_workflows.md:135]:
```suggestion
Once you have [set up the Jira integration][7], you can create cases from three areas within Test Visibility. Creating cases opens Jira issues that are both associated with your tests:
```

File: content/en/tests/developer_workflows.md [hunk 2]
```diff
@@ -122,6 +126,38 @@ You can enable test summaries in pull requests with the following steps:
 
 Comments only appear on pull requests that were opened before the test run and that have run at least one test for an enabled test service.
 
+## Create Jira issues
```
Comment (Aaron-9900) [inline content/en/tests/developer_workflows.md:129]: Two notes:
1. Maybe we could put this closer to GitHub issues?
2. When I demoed this to my team, the first questions was "does this stay in sync if I change the status of a JIRA issue? And the answer is yes. I think that is quite powerful and we might want to mention it here.

**Review by kassenq — APPROVED** [review]

**Review by Aaron-9900 — APPROVED** [review]

File: content/en/tests/developer_workflows.md [hunk 3]
```diff
@@ -68,6 +72,42 @@ You also have the option to copy an issue description in Markdown for pasting te
 Below is what a pre-filled GitHub issue might look like:
 {{< img src="ci/prefilled_github_issue.png" alt="Pre-filled GitHub issue" style="width:80%;">}}
 
+## Create Jira issues
+
+With [Case Management][8], you can create and open pre-filled Jira issues with relevant context into your tests as well as deep links back to Datadog for more streamlined debugging workflows. Creating issues directly from Test Visibility can help you track and maintain accountability for test failures and flaky tests.
```
Comment (drichards-87) [inline content/en/tests/developer_workflows.md:77]:
```suggestion
With [Case Management][8], you can create and open pre-filled Jira issues that contain relevant context related to your tests, as well as deep links back to Datadog for more streamlined debugging workflows. Creating issues directly from Test Visibility can help you track and maintain accountability for test failures and flaky tests. 
```

File: content/en/tests/developer_workflows.md [hunk 4]
```diff
@@ -68,6 +72,42 @@ You also have the option to copy an issue description in Markdown for pasting te
...
 
+## Create Jira issues
+
+With [Case Management][8], you can create and open pre-filled Jira issues with relevant context into your tests as well as deep links back to Datadog for more streamlined debugging workflows. Creating issues directly from Test Visibility can help you track and maintain accountability for test failures and flaky tests. 
+
+When you update the status of a Jira issue, the status in Case Management updates and reflects the latest case status.
+
+### In-app entry points
+
+Once you have [set up the Jira integration][7], you can create cases from three areas within Test Visibility:
```
Comment (drichards-87) [inline content/en/tests/developer_workflows.md:83]:
```suggestion
After you have [set up the Jira integration][7], you can create cases from three areas within Test Visibility:
```

**Review by drichards-87 — APPROVED** [review]
Left a couple of suggestions and approved the PR.

**Pull request merged**